annotation/tests/test_annotations.py

```diff
@@ -83,3 +172,142 @@ def test_row_to_dict(row, expected_dictionary):
     result = row_to_dict(row)
     print(result)
     assert result == expected_dictionary
+
+
+@pytest.mark.unittest
+@pytest.mark.parametrize(
+    [
+        "s3_prefix",
+        "bucket_name",
+        "expected_string",
+    ],
+    [
+        (
+            "S3_test",
+            "bucket_test",
+            "S3_test-bucket_test",
+        ),
+        (
+            None,
+            "bucket_test",
+            "bucket_test",
+        ),
+    ],
```

Comment:
Small improvement - use tuples instead of lists. Generally, if collection won't change in its lifetime - use tuple

Reply:
Done